optimade/server/tests/test_more_structures.json

```diff
@@ -1,30 +1,30 @@
-{
-  "id": "db/1234567",
+[{
+  "task_id": "db_1234567",
```

Comment:
This should be changed back to include the slash, since this is a valid `id` and should be tested.